scripts/update_proto.sh

```diff
@@ -13,8 +13,12 @@ then
     exit 1
 fi
 
-brew upgrade protobuf
-brew upgrade swift-protobuf
+if command -v brew &> /dev/null; then
```

Comment:
I don't use homebrew and this kept biting me, so I checked that homebrew exists before running these commands.